### ISSUE TITLE
Update Repo

### ISSUE THREAD

**Issue by Cubiking** [mizlan/AutofillDuo#4]
The repo is currently out of date? As the firefox extension.add-on is updated very recently.

**Comment by mizlan** [comment]
No, it shows it was updated Jan 31, 2024. Also, the add-on no longer works because Duo changed something internally which breaks the existing flow for adding a device. I will archive this repository for now

**Comment by mizlan** [comment]
I take that back; there is a fix

**Comment by mizlan** [comment]
Okay I take that back I already *implemented* the fix (described in https://github.com/WillForan/duo-hotp/pull/1). Does the extension not work? I unfortunately do not use it anymore

**Comment by Cubiking** [comment]
I think it still works. What I want to say is that I hope the repo is up to date so that I do not have to worry about the extension potentially spying me.

**Comment by mizlan** [comment]
I can confirm the last Firefox upload of AutofillDuo is in fact my upload. Beyond that you will need to trust me.

**Comment by mizlan** [comment]
Please link the addon that you say was updated very recently..
